java-frontend/src/main/java/org/sonar/plugins/java/api/semantic/Symbol.java

```diff
@@ -215,6 +215,31 @@ interface MethodSymbol extends Symbol {
      * @return the signature of the method, as String
      */
     String signature();
+
+    /**
+     * @return true if the method symbol is overridable.
+     */
+    boolean isOverridable();
+
+    /**
+     * @return true if the method has type parameters.
+     */
+    boolean isParametrizedMethod();
+
+    /**
+     * @return true if the method has a default implementation.
+     */
+    boolean isDefaultMethod();
+
+    /**
+     * @return true if the method is synchronized
```

Comment:
Let's keep it standardized. Either put or remove the `.` at the end of each sentence